sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/vcpkg.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
```suggestion
# Copyright (c) Microsoft Corporation.
# Licensed under the MIT License.

{
```

Reply:
Unfortunately vcpkg and comments don't play well with each other.